internal/kojiapi/api/openapi.yml

```diff
@@ -87,6 +87,36 @@ paths:
             text/plain:
               schema:
                 type: string
+  '/compose/{id}/manifest':
```

Comment:
This might be nitpicky, but should this not be `manifests` now? At least it should be documented as such, I think.

Which order are the manifests returned in?

Reply:
You're right.  I updated the description to mention it returns multiples, but the path and summary should be pluralised too.

The order of the manifests _should_ be the same order as when submitted.  The order should be stable for the `fsjobqueue` since dependencies are defined in an array in memory and on disk.  This will likely become an issue with the `dbjobqueue` ([see relevant comment](https://github.com/osbuild/osbuild-composer/discussions/1149#discussioncomment-275388)).

Logs use the same logic (and will have the same issue).

Reply:
Yeah I was wondering about how we handle logs. Thanks!